homepage/locales/de.ts

```diff
@@ -0,0 +1,83 @@
+export default {
+  translation: {
+    common: {
+      openInBrowser: 'Im Browser öffnen',
+      downloadApp: 'App herunterladen',
+      fileSystemBasedStorage: 'Lokaler Speicher',
+      comingSoon: 'Demnächst',
+      webApp: 'Web App',
+      desktopApp: 'Desktop App',
+      mobileApp: 'Mobile App',
+    },
+    header: {
+      forTeams: 'Für Teams',
+    },
+    hero: {
+      title: 'Boost Happiness, Productivity and Creativity.',
+      subtitle:
+        "Eine intuitive und stylishe Markdown Notiz-App für Entwickler. Vollständig Open-Source.",
+    },
+    boostHub: {
+      title: 'Boost Note für Teams',
+      description1:
+        'Wir veröffentlichen das einfache und speziell für Entwickler entworfene Wiki namens "Boost Hub" im Juni 2020!',
+      description2:
+        'Boost Hub soll ein gemeinschaftliches Wiki-Tool für Teams sein, um die Verfügbarkeit und Suchmöglichkeiten von Informationen sowohl von Erstanbietern als auch von Dritten zu zentralisieren und zu erweitern.',
+    },
+    features: {
+      title: 'Funktionen',
+      cloudStorage: 'Cloud-Speicher',
+      cloudStorageDescription:
+        'Notizen in einem Cloud-Speicher werden sicher gespeichert und sind von anderen Geräten aus zugänglich.',
+      multiplePlatforms: 'Mehrere Plattformen',
+      multiplePlatformsDescription:
+        'Die Boost Note App ist für Browser, PCs und Mobilgeräte erhältlich.',
+      syntaxHighlight: 'Syntax Highlight',
+      syntaxHighlightDescription:
+        'Boost Note unterstützt Syntax-Highlighting von mehr als 100 Programmiersprachen.',
+      mathEquations: 'Mathematische Formeln',
+      mathEquationsDescription:
+        'Boost Note unterstützt Mathe-Blöcke. Mit Hilfe der LaTeX-Syntax kann man in den Blöcken mathematische Formeln aufschreiben.',
```

Comment:
Mit Boost Note können mathematische Formeln mühelos mit LaTeX direkt in die Markdown Notiz eingebettet werden.

(this is a free translation, I provided same change for the english version)